qiita_pet/util.py

```diff
@@ -51,8 +51,13 @@ def clean_str(item):
     -------
     str
         cleaned string
+
+    Notes
+    -----
+    This function removes colons and periods from strings, and replaces spaces
+    with underscores.
     """
-    return str(item).replace(" ", "_").replace(":", "")
+    return str(item).replace(" ", "_").replace(":", "").replace(".", "")
```

Comment:
If `clean_str` is used regularly, then a more efficient replace should be done. Either a regex or translation table